content/SCALE/SCALETutorials/Shares/SMB/AddSMBShares.md

```diff
@@ -90,7 +95,7 @@ If an LDAP server is configured, select the server and click **Edit** to display
 If not configured, click **Configure LDAP** to display the **LDAP** configuration screen.
 Click **Advanced Options** and select **Samba Schema (DEPRECATED - see the help text**. 
 {{< hint warning >}}
-Only set LDAP authentication for SMB share is required and the LDAP server is configured with Samba attributes.
+Only setting LDAP authentication for SMB share is required, and configure the LDAP server with Samba attributes.
```

Comment:
yes it could be, but that sentence didn't make sense to me even after reading several times. Is "if" the correct word here? or is it "is"? I don't know.